.pre-commit-config.yaml

```diff
@@ -0,0 +1,62 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0  # Use the ref you want to point at
+    hooks:
+    -   id: trailing-whitespace
+    -   id: no-commit-to-branch
+        args: ['--branch', 'master']
```

Comment:
The default branch in this repo is `main`, not `master`. That's what needs to be protected.
```suggestion
        args: [--branch, main]
```

Reply:
Resolved.